src/views/AdminEventLog.vue

```diff
@@ -53,9 +54,10 @@
         >
             <template v-slot:expanded-item="{ headers, item }">
                 <td :colspan="headers.length">
-                  <pre>Data: {{item.representation | pretty}}</pre>
                   <pre>User ID: {{ item.userId }}</pre>
-                  <pre>Client ID: {{ item.clientId }}</pre>
+                  <pre>Application ID: {{ item.clientId }}</pre>
+                  <pre>Data: {{item.representation | pretty}}</pre>
+                  <pre>Administrator: {{item.authDetails}}</pre>
```

Comment:
No one is asking for this, but I think it's necessary to be useful. Like, the say they want the administrator's full name and the user's username displayed, but surely someone would want user IDs too. Anyhow, just put it all in "Details" until subsequent stores can clean up the presentation.